config/rbac/role.yaml

```diff
@@ -39,6 +39,18 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - ""
+  resources:
+  - namespaces
+  verbs:
+  - create
```

Comment:
Do we need create, delete? I think we would just need get, list and watch for this feature.